foundations/foundation-vertx/src/main/java/org/apache/servicecomb/foundation/vertx/server/TcpServer.java

```diff
@@ -67,19 +61,16 @@ public void init(Vertx vertx, String sslKey, AsyncResultCallback<InetSocketAddre
     }
 
     netServer.connectHandler(netSocket -> {
-      int connectedCount = connectedCounter.incrementAndGet();
+      int connectedCount = ((SCBTCPMetrics) ((NetSocketImpl) netSocket).metrics()).getConnectedCounter().get();
```

Comment:
why we can get the SCBTCPMetrics instance?

SCBTCPMetrics instance is for VertxMetrics
but netSocket.metrics() is TcpMetrics instance that returned by SCBTCPMetrics .createMetrics, but you returned Void
so, i can not understand.

Reply:
Fixed, return SCBSocketMetrics